cmake/modules/boards.cmake

```diff
@@ -21,13 +21,16 @@
 # Outcome:
 # The following variables will be defined when this CMake module completes:
 #
-# - BOARD:                Board, without revision field.
-# - BOARD_REVISION:       Board revision
-# - BOARD_DIR:            Board directory with the implementation for selected board
-# - ARCH_DIR:             Arch dir for extracted from selected board
-# - BOARD_ROOT:           BOARD_ROOT with ZEPHYR_BASE appended
-# - BOARD_EXTENSION_DIRS: List of board extension directories (If
-#                         BOARD_EXTENSIONS is not explicitly disabled)
+# - BOARD:                      Board, without revision field.
+# - BOARD_REVISION:             Board revision
+# - BOARD_QUALIFIERS:           Board qualifiers
+# - BOARD_QUALIFIERS_FILE_NAME: Board qualifiers in file name format where slashes have been
+#                               replaced with underscores
```

Comment:
Please use `NORMALIZED_BOARD_QUALIFIERS` as normilized was the decided terminology for HWMv2 when replacing `/` with `_`.

Reply:
Fixed